ECS_MIGRATION.md

```diff
@@ -0,0 +1,93 @@
+# Overview
+
+This document establishes some basic guidelines and practices for how we can continue to maintain our current cloud infrastructure while adding new reosurces to support the brand new cloud in the DOT ECS infrastructure.
+
+# Webportal-specific guidance
+
+Webportal deploys are already composed of two parts:
+
+- `prod_deploy.sh` - Pushes the angular files into a target S3 bucket for static asset hosting
+- `python3 deploy.py --environment production` - Uses Chalice to push our API Gateways and lambdas into AWS
+
+I am proposing we add a third step:
+- `terraform apply` which will deploy any non-chalice resources into AWS.
```

Comment:
Small thing but can we put the full init and apply here? Not 100% if this document is meant to be long-lasting, but if it is, I'd want both commands here.

Reply:
Yeah - I'll add it once I actually put one in. This was more for concepts before I start coding.